src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/common/actions/NotifyFlowStatsOnNewPathsAction.java

```diff
@@ -44,9 +44,9 @@ protected void perform(S from, S to, E event, C context, T stateMachine) {
                 .forEach(flowPath -> {
                     Flow flow = flowPath.getFlow();
                     UpdateFlowPathInfo pathInfo = new UpdateFlowPathInfo(
```

Comment:
wouldn't it be nice to have a builder with meaningful method names? It's difficult to track on review the correct position of arguments, especially when there are multiple arguments of the same type.

Reply:
In general - yes. But here I just added one parameter and didn't won't to refactor code to use builder here. 
Also one good thing in constructors - if you will add new param in constructor you will must add it into all constructor calls. Otherwise there will be complication error. In case of builder - you will have null values and NPE in runtime